torch/distributed/rpc.py

```diff
@@ -76,14 +76,14 @@ def _init_rpc(backend=RpcBackend.PROCESS_GROUP,
         # TODO: add try-except and destroy _agent in all processes if any fails.
         _agent = ProcessGroupAgent(self_name, group, num_send_recv_threads)
         init_rref_context(_agent)
-    elif is_backend_registered(rpc_backend):
```

Comment:
Was this code covered by tests earlier? Why didn't we see some sort of failure since `rpc_backend` wasn't initialized anywhere?

Reply:
This wasn't covered by any tests, just added one for invalid backend.

Reply:
My bad, didn't spot this in last review.

Reply:
Don't feel bad Shen, it was my diff, I am too used to letting c/c++ compiler catching undeclared variables, getting used to python.